doc/index.rst

```diff
@@ -13,6 +13,7 @@ You can browse the content by referring to the following pages:
    :caption: Subpages:
 
    about_sidewalk.rst
+   compatibility_matrix.rst
```

Comment:
consider to move it down, near release notes

Reply:
I guess that depends on when would a user need this information. If it's not required at the very beginning of setting everything up it can be moved down. Is that the case?

Reply:
Hmm, I though at the first time developers needs to set up, build and test the sdk. And when is works, they looks on compatibility matrix and release notes when it goes to final product. This matrix is not for getting started, but for the productization

Reply:
But is is also ok for me to left it as is

Reply:
Makes sense - thanks for the explanation :) will move it to right above the release notes section